docs/website/docs/building-from-source/getting-started.md

```diff
@@ -343,6 +343,11 @@ cmake -G Ninja -B ../iree-build/ \
 
 # Build
 cmake --build ../iree-build/
+
+# Install
+cd ../iree-build/
+(cd compiler && pip install -e .)
+(cd runtime && pip install -e .)
```

Comment:
I think this would be an alternative to the `source ../iree-build/.env && export PYTHONPATH` step below. @stellaraccident what are the tradeoffs of each approach / should we be recommending one over the other, or both depending on the use case?